talk/objectorientation/objectsclasses.tex

```diff
@@ -108,21 +108,26 @@
 
 \begin{frame}[fragile]
   \frametitlecpp[98]{{\ttfamily this} keyword}
-  \begin{block}{}
+  \begin{block}{How to know an object's address?}
     \begin{itemize}
-    \item {\ttfamily this} is a hidden parameter to all class methods
-    \item it points to the current object
-    \item so it is of type {\ttfamily T*} in the methods of class {\ttfamily T}
+    \item Sometimes we need to know where our class data is located
+    \item For example to implement operators, see later
+    \item All class methods can use the keyword \mintinline{cpp}{this}
+      \begin{itemize}
+        \item It returns the address of the current object
+        \item Its type is \mintinline{cpp}{T*} in the methods of a struct/class {\ttfamily T}
```

Comment:
Maybe mention that a ref on the current object can be obtained by `*this`, as it's actually more useful than the address itself, which nobody cares about.